lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -516,8 +566,14 @@ export class UserAgentApplication {
                 throw ClientAuthError.createInvalidInteractionTypeError();
             }
 
-            // prompt user for interaction
-            this.navigateWindow(urlNavigate, popUpWindow);
+            // IFRAMEDAPPS: if we are redirecting in an iframe, post a message to the topFrame
```

Comment:
Put this in helper.

Reply:
I don't think it adds any value here except too much variable passing.